spec/controllers/early_bird_dashboard_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe EarlyBirdDashboardController, type: :controller do
+  describe 'Get index' do
+    let!(:pq) { create(:pq) }
```

Comment:
If you don't reference something in the test but you need it to exist you can create it in a before block:
```suggestion
    before { create(:pq) }
```